dashboard/app/models/peer_review.rb

```diff
@@ -41,6 +41,40 @@ class PeerReview < ActiveRecord::Base
     escalated: 2
   }
 
+  def self.pull_review_from_pool(script, user)
+    # Find the first review such that meets these criteria
+    # Review is for this script
+    # I am not the submitter X
+    # Review status is nil
+    # Reviewer is nil or it has been assigned for more than a day
+    # Reviewer is not currently reviewing this level source
```

Comment:
This will require us to require users to get their focus areas before doing peer reviews. Which I'm fine with but is worth calling out